### PR TITLE
fix histogram toolbar was not showing up

### PR DESCRIPTION
**Pull request by sfeng88**
Release 2.2: https://github.com/v5analytics/visallo/pull/706, https://github.com/v5analytics/visallo/pull/715

### CODE REVIEW

File: web/war/src/main/webapp/js/util/acl.js [hunk 1]
```diff
@@ -22,18 +22,24 @@ define([
...
-                        name: elementPropertyAcl.name,
-                        key: elementPropertyAcl.keys || null
-                    });
-                    if (matches.length === 0) {
-                        propertyAcls.push(elementPropertyAcl);
-                    } else {
-                        _.each(matches, function(r) {
-                            _.extend(r, elementPropertyAcl);
+                var elements = element;
+                if (!(element instanceof Array)) {
```
Comment (joeybrk372) [inline web/war/src/main/webapp/js/util/acl.js]: This is checking the old to modify the new. Can be simplified to `const elements = element instanceof Array ? element : [element];`

File: web/war/src/main/webapp/js/util/acl.js [hunk 2]
```diff
@@ -22,18 +22,24 @@ define([
...
+                    elements = [element];
+                }
+                _.each(elements, function (e) {
```
Comment (joeybrk372) [inline web/war/src/main/webapp/js/util/acl.js]: prefer built-ins `elements.forEach((e) => {  `

File: web/war/src/main/webapp/js/util/acl.js (hunk 1 above)
Comment (jharwig) [inline web/war/src/main/webapp/js/util/acl.js]: Also use `Array.isArray`

**Comment by sfeng88** [comment]
@jharwig @joeybrk372 fixed on both PRs.

**Comment by rygim** [comment]
Can we merge this into master?

**Pull request merged**